CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 - [#5691](https://github.com/influxdata/influxdb/pull/5691): Remove associated shard data when retention policies are dropped.
 - [#5758](https://github.com/influxdata/influxdb/pull/5758): TSM engine stats for cache, WAL, and filestore. Thanks @jonseymour
 - [#5844](https://github.com/influxdata/influxdb/pull/5844): Tag TSM engine stats with database and retention policy
+- [#5902](https://github.com/influxdata/influxdb/pull/5902): Refactor query engine slice function to use a common interface for integration with Kapacitor.
```

Comment:
I don't think there's any need to add a changelog. You're modifying code that's never been part of a release. I think this commit fits in with the general changelog of "query engine was completely refactored".